trampoline/build-trampoline.py

```diff
@@ -0,0 +1,58 @@
+import subprocess
+from pathlib import Path
+import argparse
+
+
+def get_default_target() -> str:
+    result = subprocess.run(["rustc", "-vV"], capture_output=True, text=True, check=True)
+    for line in result.stdout.splitlines():
```

Comment:
maybe we could use directly a regex ? we could simplify a little the code in this way
(like this)
```host:\s+([^\s]+)```